src/Form/RedirectForm.php

```diff
@@ -133,7 +117,7 @@ public function validate(array $form, FormStateInterface $form_state) {
     $parsed_url = UrlHelper::parse(trim($source['path']));
     $path = isset($parsed_url['path']) ? $parsed_url['path'] : NULL;
     $query = isset($parsed_url['query']) ? $parsed_url['query'] : NULL;
-    $hash = Redirect::generateHash($path, $query, $form_state->getValue('language'));
+    $hash = Redirect::generateHash($path, $query, $form_state->getValue('language')[0]['value']);
```

Comment:
Which is an important point. If language is disabled, then that code might now break. Instead of changing the existing test and adding the language dependencies there, can you add a subclass that enables the additional dependencies, then it runs the same tests...